cscs-checks/prgenv/ssh_environ_check.py

```diff
@@ -0,0 +1,18 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+@rfm.simple_test
+class SSHLoginEnvCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        super().__init__()
+        self.descr = 'Test the environment when accessed remotely over SSH'
```

Comment:
The test name and description are not very informative. I would vote to have this test parameterised, with the individual env variables that we want to check. I would extend the variables to `$SCRATCH`, `$XDG_DATA_DIRS`, `$XDG_RUNTIME_DIR`, `$MODULEPATH`, `$MODULESHOME` and finally on Cray we should have `CRAY_CPU_TARGET`, `CRAYPE_NETWORK_TARGET` and `PE_PRODUCT_LIST`.

This way, I would vote to have the test description to **check if %(env_var)s is defined when accessed remotely over SSH**